### ISSUE TITLE
Inventory Unreserved to a different location from where it was originally reserved.

### ISSUE THREAD

**Issue by kbower** [kbower/tickettest1#1469]
_Imported from trac ticket 1469.
Created by **mariam**
Opened in trac: Feb 21, 2012, 2:36:52 PM
Last modified in trac: Feb 23, 2012, 7:47:42 PM_
***
_by mariam:_
See also #1467

Kent,

Different scenario:

The order reserved an item from 03-CASDGN at 13:47:15 and unreserved it to 00-WHSE at the same time.

This has happened a lot before.

The inventory team typically finds them days later. The example I sent you before and this one are the recent ones.

Mariam

```
DEL_DOC_NUM	QTY	TRN_DT	        TIME	        OLD STE	OLD LOC NEW STE	NEW LOC ITT
0218203RO99	1	02/18/12	01:47:15 PM	 	 	0	WHSE	UNR
0218203RO99	1	02/18/12	01:47:15 PM	3	CASDGN	 	 	OOR
0218203RO99	1	02/18/12	01:49:52 PM	 	 	0	WHSE	UNR
0218203RO99	1	02/18/12	01:46:47 PM	0	WHSE	 	 	OOR
```

**Comment by kbower** [comment]
_**kbower** trac comment on Feb 21, 2012, 3:36:12 PM:_

```
SQL>select CREATEDATE, SOURCE, TYPE, PRODUCTID, QTY, OLDSITEID, OLDLOCATIONID, NEWSITEID, NEWLOCATIONID from inventorytrans where orderid = '0218203RO99' order by 1;

CREATEDATE           SOURC TYP PRODUCTID        QTY OL OLDLOC NE NEWLOC
-------------------- ----- --- --------- ---------- -- ------ -- ------
18-FEB-2012 13:43:24 APP   OOR 601731931          1 00 WHSE
18-FEB-2012 13:43:24 APP   OOR 602731932          1 00 WHSE
18-FEB-2012 13:43:25 APP   OOR 603731933          1 00 WHSE
18-FEB-2012 13:43:25 APP   OOR 614731936          1 00 WHSE
18-FEB-2012 13:43:25 APP   OOR 614731936          1 00 WHSE
18-FEB-2012 13:43:25 APP   OOR 614731936          1 00 WHSE
18-FEB-2012 13:43:25 APP   OOR 614731936          1 00 WHSE
18-FEB-2012 13:47:14 SOM   UNR 601731931          1           00 WHSE
18-FEB-2012 13:47:15 SOM   OOR 601731931          1 03 CASDGN
18-FEB-2012 13:47:15 SOM   UNR 602731932          1           00 WHSE
18-FEB-2012 13:47:15 SOM   OOR 602731932          1 03 CASDGN
18-FEB-2012 13:47:15 SOM   UNR 603731933          1           00 WHSE
18-FEB-2012 13:47:15 SOM   OOR 603731933          1 03 CASDGN
18-FEB-2012 13:47:15 SOM   UNR 614731936          1           00 WHSE
18-FEB-2012 13:47:15 SOM   OOR 614731936          1 03 CASDGN
18-FEB-2012 13:47:15 SOM   UNR 614731936          1           00 WHSE
18-FEB-2012 13:47:15 SOM   OOR 614731936          1 03 CASDGN
18-FEB-2012 13:47:16 SOM   UNR 614731936          1           00 WHSE
18-FEB-2012 13:47:16 SOM   OOR 614731936          1 03 CASDGN
18-FEB-2012 13:47:16 SOM   UNR 614731936          1           00 WHSE
18-FEB-2012 13:47:16 SOM   OOR 614731936          1 03 CASDGN
18-FEB-2012 13:49:52 SOM   UNR 601731931          1           00 WHSE
18-FEB-2012 13:49:52 SOM   UNR 602731932          1           00 WHSE
18-FEB-2012 13:49:52 SOM   UNR 603731933          1           00 WHSE
18-FEB-2012 13:49:52 SOM   UNR 614731936          1           00 WHSE
18-FEB-2012 13:49:52 SOM   UNR 614731936          1           00 WHSE
18-FEB-2012 13:49:52 SOM   UNR 614731936          1           00 WHSE
18-FEB-2012 13:49:53 SOM   UNR 614731936          1           00 WHSE

28 rows selected.
```

**Comment by kbower** [comment]
_**kbower** trac comment on Feb 23, 2012, 7:47:42 PM:_

(Changeset [[2311]](../commit/7e5d2a5a201d97de3a9e6d8d322c9cba301104b0))
 * Fix concurrent change problem so we can detect if another process (e.g. GERS) has changed the data we are also editing.
   * Fixes #1467 and #1469 (Inventory creation/deletion/reservation/unreservation problems occuring when Order is edited in GERS and open in Loft)

**Comment by kbower** [comment]
_**kbower** trac status change on Feb 23, 2012, 7:47:42 PM:_ **closed**
